test-cases/gutenberg/group.md

```diff
@@ -107,21 +107,18 @@ Total space from screen edge to content should equals `16px` in each state (incl
 
 ##### TC007
 
-### Nested selection cause applying dimmed style on the rest of blocks
+### Nested selection do not cause applying dimmed style on the rest of blocks
```

Comment:
We can remove this test case, let's also remove it from the sanity-tests.md file. (No need to update the TC numbers for now)

Reply:
fixed